generation_config.yaml

```diff
@@ -0,0 +1,60 @@
+gapic_generator_version: 2.40.1-SNAPSHOT
```

Comment:
How do we plan to update this? We probably want a different strategy, like updating it with release-please, also not triggering regeneration if `gapic_generator_version` is the only parameter updated.

Reply:
> We probably want a different strategy, like updating it with release-please

Yes, I think release-please should update this value.

> also not triggering regeneration if gapic_generator_version is the only parameter updated

We only have two libraries in this repo, is this an overkill?